src/cswinrt/strings/InterfaceEntries.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Windows.Foundation;
+using static System.Runtime.InteropServices.ComWrappers;
+
+namespace WindowsRuntime.InteropServices;
+
+[StructLayout(LayoutKind.Sequential)]
+internal struct InterfaceEntries
```

Comment:
Should this be named something like `ReferenceInterfaceEntries` or something? That is, this set of interface entries is only valid for some `IReference<T>` value, not for any type.